util.sh

```diff
@@ -9,9 +9,17 @@ main() {
     echo "Tarring the ball..."
     pushd .. && tar -cvf phcode42v2/phcode42v2.tgz -X phcode42v2/exclude_files.txt phcode42v2/* && popd
     echo "Moving to remote..."
-    sshpass -p "phantom" scp phcode42v2.tgz "phantom@${PHANTOM_VM_IP_ADDR}":/home/phantom
+    scp phcode42v2.tgz "phantom@${PHANTOM_VM_IP_ADDR}":/home/phantom
     echo "Untarring on remote..."
-    sshpass -p "phantom" ssh phantom@10.0.0.41 "rm -rf phcode42v2 && tar -xvf phcode42v2.tgz && cd phcode42v2 && phenv python /opt/phantom/bin/compile_app.pyc -i"
+    ssh "phantom@${PHANTOM_VM_IP_ADDR}" "rm -rf phcode42v2 && tar -xvf phcode42v2.tgz && cd phcode42v2 && phenv python /opt/phantom/bin/compile_app.pyc -i"
+    ;;
+  deploy-bypass)
```

Comment:
do we need both of these deploy options? It seems like the second one is just better (though we should document the need for `sshpass` somewhere).

If we _do_ want both, there's enough in common between the two commands here that some stuff should probably be combined into shared functions